app/renderer/components/preferences/payment/addFundsDialog/addFundsDialogFooter.js

```diff
@@ -0,0 +1,76 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+
+// Components
+const ImmutableComponent = require('../../../immutableComponent')
+const BrowserButton = require('../../../common/browserButton')
+
+// Styles
+const {StyleSheet, css} = require('aphrodite')
+const upholdLogo = require('../../../../../extensions/brave/img/ledger/uphold-logo.png')
+
+class AddFundsDialogFooter extends ImmutableComponent {
+  render () {
+    return (
+      <section className={css(styles.addFundsFooter)} data-test-id='AddFundsDialogFooter'>
+        <div className={css(styles.addFundsFooter__start)}>
+          <img src={upholdLogo}
+            className={css(styles.addFundsFooter__init__uphold_logo)}
```

Comment:
As long as there is not another common element, why wouldn't you change this into `styles.addFundsFooter__start__uphold_logo`?